local-cli/generator-android/templates/src/app/build.gradle

```diff
@@ -63,12 +67,36 @@ android {
             abiFilters "armeabi-v7a", "x86"
         }
     }
+    splits {
+        abi {
+            enable true
+            reset()
```

Comment:
> reset(): reset the list of ABIs to be included to an empty string (this allows, in conjunctions with include, to indicate which one to use rather than which ones to ignore)

From http://tools.android.com/tech-docs/new-build-system/user-guide/apk-splits